lumigator/python/mzai/sdk/tests/helpers.py

```diff
@@ -0,0 +1,24 @@
+import unittest.mock as mock
+import importlib.resources
+import json
+from pathlib import Path
+from importlib.resources.abc import Traversable
+
+
+LUMI_HOST = "localhost"
+
+
+def load_json(datafile: Traversable) -> str:
```

Comment:
same suggestion as in other prs around Traversable, datafile can just be a string or Path